2023-risks/Draft_M3: Inadequate Supply Chain Security.md

```diff
@@ -0,0 +1,97 @@
+---
+
+layout: col-sidebar
+title: "M3: Inadequate Supply Chain Security"
+---
+
+# Threat Agents
+
+**Application Specific**
+
+An attacker can manipulate application functionality by exploiting vulnerabilities in the mobile app supply chain. For example, an attacker can insert malicious code into the mobile app's codebase or modify the code during the build process to introduce backdoors, spyware, or other malicious code.
+
+This can allow the attacker to steal data, spy on users, or take control of the mobile device. Moreover, an attacker can exploit vulnerabilities in third-party software libraries, SDKs, vendors, or hardcoded credentials to gain access to the mobile app or the backend servers. 
+
+This can lead to unauthorized data access or manipulation, denial of service, or complete takeover of the mobile app or device.
+
+# Attack Vectors
+
+**Exploitability MEDIUM**
+
+There are multiple ways to exploit Inadequate Supply Chain vulnerability for example- an insider threat agent or an attacker can inject a malicious code during the development phase of the app, then they can compromise the app signing keys or certificates to sign malicious code as trusted. 
+
+Another way, a threat agent can exploit vulnerability in third party libraries or components used in the app.
+
+# Security Weakness
+
+**Prevalence COMMON** </br>
+**Detactibility EASY**
```

Comment:
Is it really easy to detect? Given the variety of possible attack vectors and the intransparency of 3rd party dependencies, I'd rather say it is hard to assess.

Reply:
done.

Reply:
Oh, there's a typo: DetEctAbility

Reply:
done